client-js/app/elements/labrad-plot.ts

```diff
@@ -811,3 +789,66 @@ function prettyNumber(value: number, min: number, max: number, res: number): str
   }
 }
 
+
+/**
+ * Get a color for the given value on a scale with the given min and max
+ * limits (zMin <= z <= zMax).
+ */
+function getColor(z: number, zMin: number, zMax: number) {
+  var index = 128;
+  if (zMin !== zMax) {
+    index = Math.floor(255 * (z - zMin) / (zMax - zMin));
```

Comment:
I think you still want to use 256 here instead of 255, or else the range of the highest color value is contracted to occur _only_ when `z == zMax`. Then you can either restore the duplication of the top color value in `COLOR_MAP[256]` or else just add a check here to see if `z == zMax` and set index to 255 in that case.